netaddr.go

```diff
@@ -564,6 +564,27 @@ func (ip IP) IsMulticast() bool {
 	return false // zero value
 }
 
+// IsInterfaceLocalMulticast reports whether ip is an interface-local multicast
```

Comment:
Seems worth noting that this only applies to IPv6 addresses in the doc comment.

Reply:
Done, thanks.